framer/Utils.coffee

```diff
@@ -76,6 +77,12 @@ Utils.nearestIncrement = (x, increment) ->
 window.requestAnimationFrame ?= window.webkitRequestAnimationFrame
 window.requestAnimationFrame ?= (f) -> Utils.delay 1/60, f
 
+Utils.animatorFor = (options) ->
```

Comment:
Also, the structure seems pretty similar to _createAnimator of Animation. Can something be reused here? (E.g., optional default class?)